src/ocamlorg_toplevel/bin/worker.ml

```diff
@@ -91,6 +88,20 @@ let stdout_buff = Buffer.create 100
 
 let stderr_buff = Buffer.create 100
 
+(* RPC function implementations *)
+
+module M = Idl.IdM (* Identity monad - server is synchronous *)
+
+module IdlM = Idl.Make (M)
```

Comment:
Can we call it `Identity` or `Rpc_identity`? It'd make the code easier to read I think